boxes/contract-only/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@aztec/contract_box",
+  "description": "My Aztec contract",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "compile": "cd src/contracts && ${AZTEC_NARGO:-aztec-nargo} compile",
+    "codegen": "${AZTEC_CLI:-aztec-cli} codegen src/contracts/target -o artifacts --ts",
+    "clean": "rm -rf ./dest .tsbuildinfo ./artifacts ./src/contracts/target",
+    "prep": "yarn clean && yarn compile && yarn codegen && tsc -b",
+    "dev": "yarn prep && webpack serve --mode development",
+    "build": "yarn prep && webpack",
+    "serve": "webpack serve --no-open --mode development",
+    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --runInBand",
```

Comment:
This command doesn't work. There isn't a jest configuration file.

Reply:
Hmm, it should pick up the jest section from `package.json`. Does it not detect it for you @critesjosh?

Reply:
That's weird indeed, I'll double check